ui/public/locales/en.json

```diff
@@ -1986,6 +1987,7 @@
 "label.start.rolling.maintenance": "Start Rolling Maintenance",
 "label.start.rolling.maintenance.payload": "Payload",
 "label.start.vlan": "Start VLAN",
+"label.start.vm": "Start VM",
```

Comment:
May not be needed, can we use the existing 'label.action.start.instance' instead?